examples/font.js

```diff
@@ -4,42 +4,32 @@
  */
 
 var Canvas = require('../lib/canvas')
-  , canvas = new Canvas(320, 320)
-  , Font = Canvas.Font
-  , ctx = canvas.getContext('2d')
   , fs = require('fs')
   , path = require("path");
 
-if (!Font) {
-	throw new Error('Need to compile with font support');
-}
-
 function fontFile(name) {
   return path.join(__dirname, '/pfennigFont/', name);
 }
 
```

Comment:
Suggest adding a comment here to explain that the file name itself is irrelevant and that the font is found based on the name of the font in the ttf file itself.